website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -613,7 +639,7 @@ mutation {
 When querying [saved queries](/docs/build/saved-queries),you can use parameters such as `where`, `limit`, `order`, `compile`, and so on. However, keep in mind that you can't access `metric` or `group_by` parameters in this context. This is because they are predetermined and fixed parameters for saved queries, and you can't change them at query time. If you would like to query more metrics or dimensions, you can build the query using the standard format.
 :::
 
-**Create query with saved queries** 
+#### Create query with saved queries
 
 This takes the same inputs as the `createQuery` mutation, but includes the field `savedQuery`. You can use this for frequently used queries.
 
```

Comment:
The Multi-hop joins section is only related to the where filter, so it would be good to move that to the same section where we describe the where filter syntax. (Or a new page if we go that route!)

Reply:
For JDBC, we use the Jinja object syntax in the group by as well as the where filter. So the syntax is the same for these objects in JDBC yes. But this snippet is relevant to both group by and where filter in JDBC.